### PR TITLE
Add vue apps build tests

### PR DESCRIPTION
**Pull request by skjnldsv**
Ready to review!

- This also fixes some dependencies for our apps
- Also fixed our updatenotifications app (cc @blizzz )
- Reduced the packages security issues to 0
- Created tests for our vue apps
  - Check compilation
  - Check if the app is already compiled to the latest's changes (in case you forgot to build)

### CODE REVIEW

File: vue-builds.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```
Comment (MorrisJobke) [inline vue-builds.sh]: Please don't add files to the root, but rather to the `build/` directory.

Comment (skjnldsv) [reply vue-builds.sh]: Sure :)

**Review by rullzer — APPROVED** [review]
Didn't do :boom: here. I like it!

**Review by MorrisJobke — APPROVED** [review]

**Comment by blizzz** [comment]
> Also fixed our updatenotifications app (cc @blizzz )

whatever it does, it creates a merge conflict in #9705 :cry: ok, looking into it again

**Comment by skjnldsv** [comment]
@blizzz Yes, I fixed broken stuff :)

**Pull request merged**